packages/frontend/src/components/FlowStep/components/DuplicateStepButton.tsx

```diff
@@ -65,9 +69,17 @@ export default function DuplicateStepButton(props: DuplicateStepButtonProps) {
   const { flow, isMobile, onDrawerOpen, setCurrentStepId } =
     useContext(EditorContext)
 
-  const [createStep] = useMutation(CREATE_STEP, { refetchQueries: [GET_FLOW] })
-
+  const [createStep] = useMutation(CREATE_STEP)
   const onDuplicateStep = useCallback(async () => {
+    const duplicateConfig = {
+      ...(step.config?.approval && {
+        approval: step.config?.approval,
```

Comment:
```suggestion
        approval: {
          branch: step.config.approval.branch,
          stepId: step.config.approval.stepId,
        },
```

current config throws error when attempting to duplicate step in the 'reject' branch. it complains about `__typename` being present in the input

Reply:
saw this was fixed in the subsequent PR by removing __typename completely